### PR TITLE
refactor(cli): generalize module specifier collection

### PR DESCRIPTION
**Pull request by caspervonb**

### CODE REVIEW

File: cli/fs_util.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 use deno_core::error::AnyError;
 use deno_core::error::Context;
 pub use deno_core::normalize_path;
+use deno_core::url::Url;
```
Comment (kitsonk) [inline cli/fs_util.rs:6]: I would prefer we use the `ModuleSpecifier` type alias when we are going to refer to something that we expect to be a module specifier.

Comment (caspervonb) [reply cli/fs_util.rs]: sgtm :)

File: cli/fs_util.rs [hunk 2]
```diff
@@ -328,4 +372,82 @@ mod tests {
...
+    // │   ├── .foo.TS
+    // │   └── README.md
+    // ├── c.tsx
+    // ├── d.jsx
+    // └── ignore
+    //     ├── g.d.ts
+    //     └── .gitignore
+
+    let t = TempDir::new().expect("tempdir fail");
+
```
Comment (bartlomieju) [inline cli/fs_util.rs:401]: Nice test

**Review by bartlomieju — APPROVED** [review]
LGTM, good cleanup

**Pull request merged**